ui/src/data-services/models/job.ts

```diff
@@ -4,56 +4,102 @@ import { STRING, translate } from 'utils/language'
 
 export type ServerJob = any // TODO: Update this type
 
+export enum JobStatus {
+  Pending = 'pending',
+  Started = 'started',
+  Success = 'success',
+  Unknown = 'unknown',
+}
+
 export class Job {
-  private readonly _job: ServerJob
+  protected readonly _job: ServerJob
 
   public constructor(job: ServerJob) {
     this._job = job
   }
 
-  get description(): string {
-    return this._job.description
+  get finishedAt(): string | undefined {
+    if (!this._job.finished_at) {
+      return
+    }
+
+    const date = new Date(this._job.finished_at)
+    const dateString = getFormatedDateString({ date })
+    const timeString = getFormatedTimeString({ date })
+
+    return `${dateString} ${timeString}`
   }
 
   get id(): string {
-    return this._job.id
+    return `${this._job.id}`
   }
 
-  get idLabel(): string {
-    return `#${this.id}`
-  }
+  get startedAt(): string | undefined {
+    if (!this._job.started_at) {
+      return
+    }
 
-  get jobStarted(): string {
-    const date = new Date(this._job.job_started)
+    const date = new Date(this._job.started_at)
     const dateString = getFormatedDateString({ date })
     const timeString = getFormatedTimeString({ date })
 
     return `${dateString} ${timeString}`
   }
 
-  get project(): string {
-    return this._job.project
+  get name(): string {
+    return this._job.name
   }
 
-  get totalImages(): number {
-    return this._job.total_images
+  get project(): string {
+    return this._job.project.name
   }
 
-  get status(): number {
-    return this._job.status
+  get status(): JobStatus {
+    return this.getStatus(this._job.status)
   }
 
   get statusDetails(): string {
-    return 'More details about the job status.'
+    // TODO: Return BE value here when available.
```

Comment:
BE could perhaps return something here? This label will show up here:

<img width="445" alt="Screenshot 2023-07-07 at 12 48 39" src="https://github.com/RolnickLab/ami-platform/assets/11680517/65ee46a7-94ba-4dd7-8acf-c1bbeb8aa321">
<img width="630" alt="Screenshot 2023-07-07 at 12 48 47" src="https://github.com/RolnickLab/ami-platform/assets/11680517/6acf865c-6168-4627-8ff4-4f6e52723546">